spec/deprecation-cop-status-bar-view-spec.coffee

```diff
@@ -38,15 +38,15 @@ describe "DeprecationCopStatusBarView", ->
 
     deprecatedMethod()
     expect(statusBarView.textContent).toBe '1 deprecation'
-    expect(statusBarView).toShow()
+    expect(statusBarView).toBeVisible()
```

Comment:
I had to change `toShow()` to `toBeVisible()` because when the view was hidden during the initialization tests were failing. Let me know if this is allright.

Nessage from failing tests:
```
DeprecationCopStatusBarView
  it increments when there are deprecated methods
    Expected element '[object HTMLDivElement]' or its descendants to show.
      at [object Object].<anonymous> (D:\Development\atom\deprecation-cop\spec\deprecation-cop-status-bar-view-spec.coffee:41:27)
      at _fulfilled (C:\Users\kubetz\AppData\Local\atom\app-1.0.0\resources\app.asar\node_modules\q\q.js:794:54)
      at self.promiseDispatch.done (C:\Users\kubetz\AppData\Local\atom\app-1.0.0\resources\app.asar\node_modules\q\q.js:823:30)
      at Promise.promise.promiseDispatch (C:\Users\kubetz\AppData\Local\atom\app-1.0.0\resources\app.asar\node_modules\q\q.js:756:13)
      at C:\Users\kubetz\AppData\Local\atom\app-1.0.0\resources\app.asar\node_modules\q\q.js:564:44
      at flush (C:\Users\kubetz\AppData\Local\atom\app-1.0.0\resources\app.asar\node_modules\q\q.js:110:17)
      at process._tickCallback (node.js:357:13)
    Expected element '[object HTMLDivElement]' or its descendants to show.
      at [object Object].<anonymous> (D:\Development\atom\deprecation-cop\spec\deprecation-cop-status-bar-view-spec.coffee:45:27)
      at _fulfilled (C:\Users\kubetz\AppData\Local\atom\app-1.0.0\resources\app.asar\node_modules\q\q.js:794:54)
      at self.promiseDispatch.done (C:\Users\kubetz\AppData\Local\atom\app-1.0.0\resources\app.asar\node_modules\q\q.js:823:30)
      at Promise.promise.promiseDispatch (C:\Users\kubetz\AppData\Local\atom\app-1.0.0\resources\app.asar\node_modules\q\q.js:756:13)
      at C:\Users\kubetz\AppData\Local\atom\app-1.0.0\resources\app.asar\node_modules\q\q.js:564:44
      at flush (C:\Users\kubetz\AppData\Local\atom\app-1.0.0\resources\app.asar\node_modules\q\q.js:110:17)
      at process._tickCallback (node.js:357:13)
    Expected element '[object HTMLDivElement]' or its descendants to show.
      at [object Object].<anonymous> (D:\Development\atom\deprecation-cop\spec\deprecation-cop-status-bar-view-spec.coffee:49:27)
      at _fulfilled (C:\Users\kubetz\AppData\Local\atom\app-1.0.0\resources\app.asar\node_modules\q\q.js:794:54)
      at self.promiseDispatch.done (C:\Users\kubetz\AppData\Local\atom\app-1.0.0\resources\app.asar\node_modules\q\q.js:823:30)
      at Promise.promise.promiseDispatch (C:\Users\kubetz\AppData\Local\atom\app-1.0.0\resources\app.asar\node_modules\q\q.js:756:13)
      at C:\Users\kubetz\AppData\Local\atom\app-1.0.0\resources\app.asar\node_modules\q\q.js:564:44
      at flush (C:\Users\kubetz\AppData\Local\atom\app-1.0.0\resources\app.asar\node_modules\q\q.js:110:17)
      at process._tickCallback (node.js:357:13)
```